src/main/java/com/google/cloud/genomics/utils/grpc/GenomicsChannel.java

```diff
@@ -26,10 +36,13 @@
 
 import javax.net.ssl.SSLException;
 
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.genomics.utils.GenomicsFactory;
+
 /**
  * A convenience class for creating gRPC channels to the Google Genomics API.
  */
-public class GenomicsChannel extends Channel {
+public class GenomicsChannel extends ManagedChannel {
```

Comment:
I wouldn't suggest you to implement `ManagedChannel`. You can keep the factory methods, but I would suggest they just return a `ManagedChannel`.